eg01-eg23/eg01-eg23-cycamore.xml

```diff
@@ -1,279 +1,303 @@
-            <commod>LWR Reprocessed</commod>
+            <commod>uox_U</commod>
             <info>
-              <buf_size>1e+100</buf_size>
+              <buf_size>1E100</buf_size>
               <efficiencies>
                 <item>
-                  <comp>Pu</comp>
-                  <eff>0.99</eff>
+                  <comp>U</comp> <eff>.998</eff>
                 </item>
               </efficiencies>
             </info>
           </item>
         </streams>
-        <throughput>1e+100</throughput>
       </Separations>
     </config>
-    <name>LWR Separation</name>
   </facility>
 
-
   <facility>
+    <name>mox_reprocessing</name>
     <config>
       <Separations>
-        <feed_commod_prefs>
-          <val>2</val>
-        </feed_commod_prefs>
-        <feed_commods>
-          <val>FR Spent Fuel</val>
-        </feed_commods>
-        <feedbuf_size>1e+100</feedbuf_size>
-        <leftover_commod>WASTE</leftover_commod>
-        <streams>
+         <feed_commods>   <val>cool_sfr_spent</val> <val>cool_mox_waste</val> </feed_commods>
+         <feed_commod_prefs> <val>2.0</val> <val>1.0</val> </feed_commod_prefs>
+         <feedbuf_size>1E100</feedbuf_size>
+         <throughput>1E100</throughput>
+         <leftover_commod>mox_reprocess_waste</leftover_commod>
+         <leftoverbuf_size>1E100</leftoverbuf_size>
+         <streams>
+          <item>
+            <commod>mox_Pu</commod>
+            <info>
+              <buf_size>1E100</buf_size>
+              <efficiencies>
+                <item>
+                  <comp>Pu</comp> <eff>.998</eff>
+                </item>
+              </efficiencies>
+            </info>
+          </item>
           <item>
-            <commod>FR Reprocessed</commod>
+            <commod>mox_U</commod>
             <info>
-              <buf_size>1e+100</buf_size>
+              <buf_size>1E100</buf_size>
               <efficiencies>
                 <item>
-                  <comp>Pu</comp>
-                  <eff>0.99</eff>
+                  <comp>U</comp> <eff>.998</eff>
                 </item>
               </efficiencies>
             </info>
           </item>
         </streams>
-        <throughput>1e+100</throughput>
       </Separations>
     </config>
-    <name>FR Reprocess</name>
   </facility>
 
-
   <facility>
     <config>
-      <Reactor>
-        <assem_size>11666.7</assem_size>
-        <cycle_time>18</cycle_time>
-        <fuel_incommods>
-          <val>LWR Fuel</val>
-        </fuel_incommods>
-        <fuel_inrecipes>
-          <val>fresh_uox</val>
-        </fuel_inrecipes>
-        <fuel_outcommods>
-          <val>LWR Spent Fuel</val>
-        </fuel_outcommods>
-        <fuel_outrecipes>
-          <val>spent_uox</val>
-        </fuel_outrecipes>
-        <fuel_prefs>
+      <FuelFab>
+        <fill_commods>
+          <val>tailings</val>
+        </fill_commods>
+        <fill_recipe>depleted_u</fill_recipe>
+        <fill_size>1e+60</fill_size>
```

Comment:
I think the 60 was just from an example I copied and pasted from, it's fixed to `1E100`.